treeherder/webapp/api/performance_data.py

```diff
@@ -819,6 +820,8 @@ def list(self, request):
                 base_stddev_pct = self._get_stddev_pct(base_avg_value, base_stddev)
                 new_stddev_pct = self._get_stddev_pct(new_avg_value, new_stddev)
 
+                sig_hash = base_sig_id if base_sig_id else new_sig_id
```

Comment:
The old logic where you get the signature hash was correct - signature id is not the same as siganture hash, although they are unique identifiers for the signatures.